dlfcn.h

```diff
@@ -55,6 +55,22 @@ extern "C" {
 /* Specifies the next object after this one that defines name. */
 #define RTLD_NEXT       ((void *)-1)
 
+#ifdef __USE_GNU
+
+#include <limits.h>
+
+/* Structure filled in by dladdr() */
+typedef struct dl_info
+{
+   const char *dli_fname;            /**< Filename of defining object */
+   void       *dli_fbase;            /**< Load address of that object */
+   const char *dli_sname;            /**< Name of nearest lower symbol */
+   void       *dli_saddr;            /**< Exact value of nearest symbol */
+} Dl_info;
+
+DLFCN_EXPORT int dladdr(const void *addr, Dl_info *info);

```

Comment:
Just cosmetic change, but I think that this function could be put in header file after all other dl-functions as it is non-standard. I think that first should be mandatory POSIX functions and then some possible extensions (and documented that they are not specified in POSIX standard).